students/Ajay_Randhawa/lesson03/Slicing.py

```diff
@@ -0,0 +1,45 @@
+def firstLast(seq):
+	""" switches the first and last values of a sequence"""
+	seq1 = seq[-1:] + seq[1:-1] + seq[:1]
```

Comment:
Looks like you're over indented in this file.  Every indentation should be 4 spaces.  Make sure you have your text editor to convert tabs to spaces.